src/pages/status.js

```diff
@@ -0,0 +1,180 @@
+import componentMetadata from '@primer/component-metadata'
+import {Box, Heading, Text, themeGet} from '@primer/components'
+import StatusLabel from '@primer/gatsby-theme-doctocat/src/components/status-label'
+import fetch from 'isomorphic-unfetch'
+import React from 'react'
+import styled from 'styled-components'
+import Layout from '../components/Layout'
+import LinkLight from '../components/LinkLight'
+
+// TODO: Make table header sticky
```

Comment:
I don't think this is easy because the `overflow: auto` on the table gives it a 'scrolling context' which breaks the sticky (see https://github.com/w3c/csswg-drafts/issues/865); one solution would be to only add `overflow: auto` on smaller screens (at least then it'd be sticky on wide) or make a mobile table view which doesn't need horizontal scrolling (probably not worth it for this).

Reply:
Ah, bummer. A sticky header would have been nice, but it's probably not worth the hacks